### ISSUE TITLE
pyapns2 requirement missing from setup.cfg

### ISSUE THREAD

**Issue by matthewh** [jleclanche/django-push-notifications#380]
This impacts the unrealsed master branch only.

pyapns2 is required for APNS support but is missing from the requirements section of setup.cfg.

Modify setup.cfg to include:

```
[options]
packages = find:
install_requires =
	apns2
	Django>=1.8
```

**Comment by jleclanche** [comment]
yep feel free to.

**Comment by matthewh** [comment]
Merged with #370.
